specification/dimensions/usageunit.md

```diff
@@ -33,18 +43,37 @@ Units listed in the following table are presented in the following format: "abbr
 
 If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning or incompatible capitalization.  An example of unit now allowed is a unit such as "calls" or "number" to signify the number of times an event occurred in this case, the unit "count" MUST be used to maintain compatibility with this specification.
 
-| TIME_UNIT        | DATA_UNIT | COUNT_UNIT   | NUMERIC PREFIXES        |
-|------------------|-----------|--------------|-------------------------|
-| yr: year         | bit: bit  | cnt: count   | Ki: kibi (2^10)         |
-| mo: month        | B: byte   |              | Mi: mebi (2^20)         |
-| wk: week         |           |              | Gi: gibi (2^30)         |
-| d: day           |           |              | Ti: tebi (2^40)         |
-| h: hour          |           |              | Pi: pebi (2^50)         |
-| min: minute      |           |              | K: kilo (10^3)          |
-| s: second        |           |              | M: mega (10^6)          |
-| ms: millisecond  |           |              | G: giga (10^9)          |
-| us: microsecond  |           |              | T: tera (10^12)         |
-| ns: nanosecond   |           |              | P: peta (10^15)         |
+The following table lists the valid units for 3 dimensions of measurement: time, data, and count.  
+
+| Time             | Data      | Count        |
+|------------------|-----------|--------------|
+| yr: year         | bit: bit  | cnt: count   |
+| mo: month        | B: byte   |              |
+| wk: week         |           |              |
+| d: day           |           |              |
+| h: hour          |           |              |
+| min: minute      |           |              |
+| s: second        |           |              |
+| ms: millisecond  |           |              |
+| us: microsecond  |           |              |
+| ns: nanosecond   |           |              |
```

Comment:
Do we really want to allow this? It feels entirely too low-level. We can represent all of these using fractions of hours or minutes. When you're talking about something this small at a large scale, it becomes entirely unreadable by humans, which breaks one of our tenants of being human readable/consumable.

Reply:
In practice, I can't think of anything that comes across in weeks or days. It's either months or hours.

Reply:
Mostly done - I dropped everything but "Day, Hour, Minute, Second" - why did I keep second?    I would rather parse 90 seconds than 1.5 minutes.  I think we should leave some flexibility on level up and one level down.     Also, from a licensing perspective, days is something I see as relevant.

Check the latest revision of the PR.